### DISCUSSION TITLE
why discord.py dosen`t surport discord.ext

### DISCUSSION THREAD

**Discussion by alpha216** [Rapptz/discord.py#6138] [Q&A] [answered]
i am using python3.8.6 and i downloaded discord.py from pip 
here is my pip install 
`aiohttp==3.6.3
async-timeout==3.0.1
attrs==20.3.0
chardet==3.0.4
discord.py==1.5.1
idna==2.10
multidict==4.7.6
yarl==1.5.1`
but when i start program `No module named 'discord.ext'; 'discord' is not a package `error comes up
Why is this happening?

**Comment by Nemika-Haj** [comment, marked as answer]
Have you tried removing and reinstalling Discord.py? Remove it using `pip uninstall discord.py` and reinstall it. If that doesn't solve the issue, try removing and adding PIP again.

**Reply by Nemika-Haj** [reply]
Adding to that check if you have the latest pip version.

**Reply by NotTani** [reply]
To add to that, check if your discord.py is on the latest version and it's not erroring out during install
